src/shared/components/case-editor/case-edit-submit/case-edit-submit.component.ts

```diff
@@ -131,13 +131,12 @@ export class CaseEditSubmitComponent implements OnInit {
   }
 
   summaryCaseField(field: CaseField): CaseField {
-    let cloneField: CaseField = Object.assign({}, field);
-
     if (null == this.editForm.get('data').get(field.id)) {
       // If not in form, return field itself
       return field;
     }
 
+    let cloneField: CaseField = Object.assign({}, field);
```

Comment:
we have a method `cloneObject` on FieldsUtils which this class is already injected with. You could make that method public and reuse it (I think to remember it was public for reusability, not sure why is private now)